docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/airgapped.md

```diff
@@ -13,14 +13,18 @@ showAsideToc: true
 ---
 
 <ul class="nav nav-tabs-alt nav-tabs-yb">
-
   <li >
     <a href="/latest/yugabyte-platform/install-yugabyte-platform/install-software/default" class="nav-link">
       <i class="fas fa-cloud"></i>
       Default
     </a>
   </li>
 
+  <li >
+    <a href="/latest/yugabyte-platform/install-yugabyte-platform/install-software/openshift" class="nav-link">
+      <i class="fas fa-cloud"></i> OpenShift </a>
```

Comment:
```suggestion
      <i class="fas fa-cubes"></i> OpenShift </a>
```

I think we should keep it consistent across different pages.

Reply:
Fixed